tuxemon/core/components/menu/save_menu.py

```diff
@@ -40,12 +40,12 @@ def draw(self):
             slot_name = "slot" + str(slot_num)
 
             # Check to see if a save exists for the current slot
-            if os.path.exists("saves/slot" + str(slot_num) + ".png"):
+            if os.path.exists(".saves/slot" + str(slot_num) + ".png"):
```

Comment:
Right now this is going to put save games under the current working directory. We should instead put these save games under the user's home directory. Here's a possible way we could handle this:

https://stackoverflow.com/questions/4028904/how-to-get-the-home-directory-in-python?answertab=votes#tab-top